BUILD.md

```diff
@@ -0,0 +1,65 @@
+# Building the project
```

Comment:
Could you please update the README as well to remove the redundant sections there and point to this BUILD.md document?